internal/conf/configuration.go

```diff
@@ -365,6 +367,15 @@ func LoadGlobal(filename string) (*GlobalConfiguration, error) {
 	} else {
 		config.SAML.PrivateKey = ""
 	}
+	if config.Sms.Provider != "" {
+		defaultTemplateName := "default_sms_template_name"
+		if config.Sms.Template == "" {
+			defaultSMSTemplate := "Your code is {{ .Code }}"
+			config.Sms.SMSTemplate = template.Must(template.New(defaultTemplateName).Parse(defaultSMSTemplate))
+		} else {
+			config.Sms.SMSTemplate = template.Must(template.New(defaultTemplateName).Parse(config.Sms.Template))
+		}
```

Comment:
AFAIK `defaultTemplateName` can just be `""` so we don't have to define constants. It's just one template.

I'd actually use the `template.New()` without `Must`, as that method will panic on error. It's best if we handle it and just use the default template while logging an error message.